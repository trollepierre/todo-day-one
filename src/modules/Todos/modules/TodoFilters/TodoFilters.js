export const TodoFilters = ({ sortByCreatedDate }) =>
  (
    <aside>
      TodoFilters
      <button onClick={sortByCreatedDate}>Inverser l'ordre de tri par date de crÃ©ation</button>
    </aside>
  )
