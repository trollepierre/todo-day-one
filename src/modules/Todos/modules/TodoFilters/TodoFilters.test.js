import { render } from '@testing-library/react'
import { TodoFilters } from './TodoFilters'

describe('TodoFilters', () => {
  it('should match snapshot', async () => {
    const {container} = render(
        <TodoFilters/>
    )
    expect(container).toMatchSnapshot()
  })

  // TODO: testing methods
})
