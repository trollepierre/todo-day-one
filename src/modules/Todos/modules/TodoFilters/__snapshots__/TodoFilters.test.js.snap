// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoFilters should match snapshot 1`] = `
<div>
  <aside>
    TodoFilters
    <button>
      Trier par date de cr√©ation
    </button>
  </aside>
</div>
`;
